content/docs/1.2.3/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support.md

```diff
@@ -21,6 +21,8 @@ Note that the snapshot controller YAML files mentioned below deploy into the `de
 > For general use, update the snapshot controller YAMLs with an appropriate **namespace** prior to installing.
 >
 > For example, on a vanilla Kubernetes cluster, update the namespace from `default` to `kube-system` prior to issuing the `kubectl create` command.
+>
+> Note that, you must use the same release version of snapshot CRDs and snapshot controller.
```

Comment:
```suggestion
> **NOTE**: Please install the same release version of snapshot CRDs and snapshot controller to ensure that the CRD version is compatible with the snapshot controller.
```

Reply:
Also, suggest moving this note on the top before the original two statements.

Reply:
I remove the **NOTE** wording to make the paragraph clear to read.